src/facade/redis_parser.cc

```diff
@@ -3,16 +3,19 @@
 //
 #include "facade/redis_parser.h"
 
+#include <absl/flags/flag.h>
 #include <absl/strings/escaping.h>
 #include <absl/strings/numbers.h>
 
 #include "base/logging.h"
 #include "core/heap_size.h"
 
+using facade::operator""_MB;
+ABSL_FLAG(int64_t, max_bulk_len, 256_MB, "Bulk length threshold for rejecting large requests");
```

Comment:
i actually prefer that we have much higher limit (2GB) by default, and then reduce it in production.
the motivation: improve "compatibility" for community use-cases, when people try out dragonfly